### ISSUE TITLE
fix is not working without fixType

### ISSUE THREAD

**Issue by frankie303** [eps1lon/eslint-focus#32]
Hey! 👋🏻 `--fix` does not work without also adding `--fix-type` as an argument so just wondering if this was a design decision or bug? If it's bug, I can create a PR please let me know. Otherwise, maybe _"Same as `eslint --fix`"_ in docs is a bit misleading since in ESLint, it should work without `--fix-type` as well. Thanks!

**Comment by eps1lon** [comment]
Right, we can add `['problem']` as the default in https://github.com/eps1lon/eslint-focus/blob/b39fbd2da44de6e68006240ce21bf92f64a0e5aa/bin/eslint-focus.js#L355. I think that mirrors how ESLint's CLI works?

**Comment by frankie303** [comment]
I think in ESLint, `fix` without `fixType` should fix all possible types. So in this case, would it make sense to update the logic here like this?
![Screenshot 2024-12-18 at 16 09 14](https://github.com/user-attachments/assets/7facc782-2fd6-46d0-82eb-218e0585be7a)

**Comment by eps1lon** [comment]
> I think in ESLint, fix without fixType should fix all possible types.

Can you check that this is indeed the case?

**Comment by frankie303** [comment]
[Seems like it](https://github.com/eslint/eslint/blob/main/lib/eslint/eslint.js#L412-L413)

**Comment by eps1lon** [comment]
Thanks for digging. Would you mind filing a PR with the necessary changes?

**Comment by frankie303** [comment]
Of course! :)
